src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -5235,5 +5236,214 @@ void test(SemanticModelOptions options)
                 Assert.Equal(PublicNullableAnnotation.None, typeInfo.Type.NullableAnnotation);
             }
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/71522")]
+        public void CollectionExpression_NestedNullability_01()
```

Comment:
>_NestedNullability [](http://example.com/codeflow?start=40&length=18)

nit: I didn't quite follow what is the "nested nullability" aspect of those tests.

Reply:
ElementNullableAnnotation is essentially nested, it is not tracked directly in the TypeWithState for an array value, for example, rather it is the ElementNullableAnnotation on the ArrayTypeSymbol.